README.md

```diff
@@ -3,21 +3,20 @@
 ![Matrix](https://img.shields.io/matrix/xtra-community:matrix.org)
 
 # xtra
-A tiny, fast, and safe actor framework. It is modelled around Actix (copyright and license [here](https://github.com/Restioson/xtra/blob/master/LICENSE-ACTIX)).
 
-For better ergonomics with xtra, try the [spaad](https://crates.io/crates/spaad) crate.
```

Comment:
If you want to mention it, please suggest a wording for what it should say @Restioson ! :)

Reply:
I think this is fine to be honest